contrib/auto-render/auto-render.js

```diff
@@ -1,7 +1,7 @@
 /* eslint no-console:0 */
 /* global katex */
 
-import splitAtDelimiters from "./splitAtDelimiters";
+const splitAtDelimiters = require("./splitAtDelimiters").default;
```

Comment:
It should be possible to mix `import` syntax with `module.exports`.

Reply:
@rrandallcainc I double checked this by reverting this line, running `make dist`, and doing `const renderMathInElement = require("./dist/contrib/auto-render.js")`.  The result was a function.  Can you change the `require` statements back to `import`s?